app/assets/stylesheets/theme_sinai/components/_si-pagination-block.scss

```diff
@@ -14,13 +14,13 @@
 
   &:hover {
     color: $white;
-    background-color: $sinai-primary;
-    border-color: $sinai-primary;
+    background-color: #5b5b5b;
```

Comment:
Could you name this new color in the sinai/color section and use that variable for these